manifests/m/Microsoft/ADKPEAddon/10.1.22000.1/Microsoft.ADKPEAddon.yaml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Microsoft.ADKPEAddon
+PackageVersion: 10.1.22000.1
+PackageName: Windows Assessment and Deployment Kit Windows Preinstallation Environment Add-ons
+Publisher: Microsoft Corporation
+License: Copyright (c) Microsoft Corporation
+Moniker: adkpe
+Tags:
+- mdt
+- adk
+- sccm
+- winpe
+ShortDescription: The Windows Preinstallation Environment addon for the Windows ADK.
+PackageUrl: https://docs.microsoft.com/en-us/windows-hardware/get-started/adk-install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.microsoft.com/download/5/5/e/55e3e34a-5708-46cd-a90d-92044c29336b/adkwinpeaddons/adkwinpesetup.exe
+  InstallerSha256: BB93457C2B7859B15454ADF0DEFE48A6A1E90C8A47AA8FE8F68F911973AC7772
+  InstallerType: exe
```

Comment:
:c
```suggestion
  InstallerType: burn
```

Reply:
How can you tell? (If you have a good way of telling outside of running wingetcreate on it I'm all ears)

Reply:
/quiet and /passive when its not MSI is a good tell
Tbh i just run wingetcreate new <url> on my VM without saving the manifest to double check